locator/topology.hh

```diff
@@ -437,11 +437,38 @@ std::ostream& operator<<(std::ostream& out, const locator::node::state& state);
 
 } // namespace std
 
+// Accepts :v format option for verbose printing
 template <>
 struct fmt::formatter<locator::node> : fmt::formatter<std::string_view> {
+    bool verbose = false;
+    constexpr auto parse(fmt::format_parse_context& ctx) {
+        auto it = ctx.begin(), end = ctx.end();
+        if (it != end) {
+            if (*it == 'v') {
+                verbose = true;
+                it++;
+            }
+            if (it != end && *it != '}') {
+                throw fmt::format_error("invalid format specifier");
+            }
+        }
+        return it;
+    }
     template <typename FormatContext>
     auto format(const locator::node& node, FormatContext& ctx) const {
-        return fmt::format_to(ctx.out(), "{}/{}", node.host_id(), node.endpoint());
+        if (!verbose) {
+            return fmt::format_to(ctx.out(), "{}/{}", node.host_id(), node.endpoint());
+        } else {
```

Comment:
I'd say the default format should include all the contents. The short format would normally be just format("{}", node.host_id()), but because we're transitioning from node ip to node id we print both. But this would be the special one.